Week2/README.md

```diff
@@ -26,22 +26,22 @@ Only watch the below chapters:
 3. Variables and Data Types
 
 
+Read up on the most important topics of next week:
 
+[Variables (var, let, const)](http://javascript.info/variables)
 
+[Basic Data types(Strings, Numbers, Arrays, Booleans)](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/values.md)
 
-- Helpful resource: http://jsbooks.revolunet.com/ (here you can find tons of free JavaScript books online)
+[Operators](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/operators.md)
 
-- The 50 best websites to Learn JavaScript: http://www.codeconquest.com/blog/top-50-websites-to-learn-javascript/
+[Special characters and their names](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/names_of_special_characters.md)
 
-:star: You can also already go through the topics of the upcoming lecture. :star:
+[Naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)
 
 
-- Intro JavaScript (What is it, where can you use it for)  
-- [Variables (var, let, const)](./../../../../fundamentals/blob/master/fundamentals/variables.md)  
-- [Basic Data types (Strings, Numbers, Arrays, Booleans)](./../../../../fundamentals/blob/master/fundamentals/values.md)  
-- [Operators](./../../../../fundamentals/blob/master/fundamentals/operators.md)  
-- [Special characters and their names](./../../../../fundamentals/blob/master/fundamentals/names_of_special_characters.md)  
-- [Naming conventions](./../../../../fundamentals/blob/master/fundamentals/naming_conventions.md)
+- Helpful resource: http://jsbooks.revolunet.com/ (here you can find tons of free JavaScript books online)
+
+- The 50 best websites to Learn JavaScript: http://www.codeconquest.com/blog/top-50-websites-to-learn-javascript/
```

Comment:
This is just confusing and doesn't provide any quality insight.

Reply:
I agree, that was already in the original document though.